code/script.js

```diff
@@ -222,104 +235,167 @@ const generateBoard = () => {
   })
 }
 
-// Randomly select a person from the characters array and set as the value of the variable called secret
+// Randomly select a person from the characters array and set as the value of the variable called secret.
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  console.log(`Secret person: ${secret.name}`);
 }
 
-// This function to start (and restart) the game
+// This function to start (and restart) the game.
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
-  charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  // Setting charactersInPlay array to be all the characters to start with.
+  charactersInPlay = CHARACTERS;
+  generateBoard(); // Invokes generateBoard function.
+  setSecret(); // Invokes setSecret function.
+  filterBtn.disabled = true; // Disables the "Find Out" button before the player has selected an option from the selection menu.
+  filterBtn.style.opacity = "0.5"; // Opacity set on the find out button.
 }
 
-// setting the currentQuestion object when you select something in the dropdown
+// Setting the currentQuestion object when you select something in the dropdown.
 const selectQuestion = () => {
-  const category = questions.options[questions.selectedIndex].parentNode.label
+  filterBtn.disabled = false; // Enables the "Find Out" button when a selection has been made.
+  filterBtn.style.opacity = "1"; // Opacity set to default on the find out button.
+  const category = questions.options[questions.selectedIndex].parentNode.label; // This variable stores what option group the question belongs to.
 
   // This variable stores what option group (category) the question belongs to.
```

Comment:
I think maybe you left the already existing comment, and wrote a new one that explains the same piece of code? Makes it a little confusing